Source/WebKit/WebProcess/WebPage/DrawingArea.h

```diff
@@ -165,6 +165,8 @@ class DrawingArea : public IPC::MessageReceiver, public WebCore::DisplayRefreshM
     virtual void adoptLayersFromDrawingArea(DrawingArea&) { }
     virtual void adoptDisplayRefreshMonitorsFromDrawingArea(DrawingArea&) { }
 
+    virtual void setNextRenderingUpdateRequiresSynchronousImageDecoding() { }
```

Comment:
If this function is going to be accessed from the base class, I'd rather have the flag also be in the base class and do not make this function virtual. For me virtual function means completely different implementation/meaning in the superclass. But  in this case the superclass is doing the obvious thing.